test/cluster/auth_cluster/test_auth_no_quorum.py

```diff
@@ -27,7 +27,7 @@ async def test_auth_no_quorum(manager: ManagerClient) -> None:
         'permissions_validity_in_ms': 0,
         'permissions_update_interval_in_ms': 0,
     }
-    servers = await manager.servers_add(3, config=config)
+    servers = await manager.servers_add(3, config=config, auto_rack_dc="dc1")
```

Comment:
I do not think this test cares about DCs, so why do we need to define any here? Could it be a default?

I think this would be good followup, no need to change it here